orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/OrientDBDialect.java

```diff
@@ -92,94 +109,285 @@ public Tuple getTuple(EntityKey key, TupleContext tupleContext) {
 	}
 
 	@Override
+	public void forEachTuple(ModelConsumer consumer, EntityKeyMetadata... entityKeyMetadatas) {
+		throw new UnsupportedOperationException( "forEachTuple!.Not supported yet." );
+	}
+
+/*	@Override
+	public List<Tuple> getTuples(EntityKey[] keys, TupleContext tupleContext) {
+		ArrayList<Tuple> tuples = new ArrayList<>( keys.length );
+		for ( EntityKey key : keys ) {
+			log.info( "getTuples:EntityKey:" + key + "; tupleContext" + tupleContext );
+			tuples.add( getTuple( key, tupleContext ) );
+		}
+		return tuples;
+	} */
```

Comment:
Note that you can alway keep this in a separate branch and include them when the time comes